src/modules/auth/auth.service.ts

```diff
@@ -34,5 +43,126 @@ export class AuthService {
         return newUser
       }
 
-     
+      async login(loginDto: LoginDto) {
+        const user = await this.prismaService.users.findUnique({
```

Comment:
Why does Auth Service need to care about fetching unique user? Move this to UsersService.